### PR TITLE
UIREQ-946 JEST/RTL test cases for RequestQueueRoute

### PR DESCRIPTION
**Pull request by KetineniM**
UIREQ-946 JEST/RTL test cases for RequestQueueRoute
URL: https://issues.folio.org/browse/UIREQ-946

### CODE REVIEW

**Review by cchitneedi — APPROVED** [review]

File: src/routes/RequestQueueRoute.test.js [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+        history={mockHistory}
+        match={mockMatch}
+        location={mockLocation}
+      />
+    );
+    expect(mockMutator.requests.reset).toHaveBeenCalledTimes(1);
+    expect(mockMutator.requests.GET).toHaveBeenCalledTimes(1);
+    expect(mockMutator.requests.GET).toHaveBeenCalledWith({
+      path: 'circulation/requests/queue/item/2',
+    });
```
Comment (artem-blazhko) [inline src/routes/RequestQueueRoute.test.js:175]: Could you create separate tests for each "expect"?

**Review by artem-blazhko — CHANGES_REQUESTED** [review]

**Review by Dmitriy-Litvinenko — COMMENTED** [review]
Could you please fix feedbacks?

File: src/routes/RequestQueueRoute.test.js [hunk 2]
```diff
@@ -0,0 +1,219 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import RequestQueueRoute from './RequestQueueRoute';
+
+jest.mock('@folio/stripes/core', () => ({
+  stripesConnect: jest.fn(Component => props => <Component {...props} />)
+}));
```
Comment (artem-blazhko) [inline src/routes/RequestQueueRoute.test.js:8]: We already have it in `test/jest/__mock__/stripesCore.mock.js` file

File: src/routes/RequestQueueRoute.test.js [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+      <RequestQueueRoute
+        resources={mockResources}
+        mutator={mockMutator}
+        history={mockHistory}
+        location={mockLocation}
+        match={mockMatch}
+      />
+    );
+
+    expect(mockMutator.requests.GET).toHaveBeenCalled();
```
Comment (artem-blazhko) [inline src/routes/RequestQueueRoute.test.js:138]: Seems that it is redundant and it is a part of the next test.

File: src/routes/RequestQueueRoute.test.js [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+        resources={mockResources}
+        mutator={mockMutator}
+        history={mockHistory}
+        location={mockLocation}
+        match={mockMatch}
+      />
+    );
+    expect(container).toBeInTheDocument();
+  });
+  it('should call setTlrSettings on mount if configs have loaded', () => {
```
Comment (artem-blazhko) [inline src/routes/RequestQueueRoute.test.js:112]: Name of the test does not fit test body.
In this test you check that `requests.GET` is called after rerender and is not called after mount.

**Review by artem-blazhko — CHANGES_REQUESTED** [review]

**Comment by Dmitriy-Litvinenko** [comment]
Could you please fix feedbacks?

File: src/routes/RequestQueueRoute.test.js (hunk 4 above)
Comment (artem-blazhko) [reply src/routes/RequestQueueRoute.test.js:97]: Name of the test does not fit test body.
In this test you check that requests.GET is called after rerender but not after the first render (mount).

**Review by Dmitriy-Litvinenko — APPROVED** [review]

**Review by artem-blazhko — APPROVED** [review]

**Pull request merged**